browser/src/control/jsdialog/Widget.ColorPicker.ts

```diff
@@ -135,15 +135,16 @@ function createColor(
 				event.preventDefault();
```

Comment:
above keyboard event listener also could be moved to util (it doesn't use widget-specific things, just our helper functions)

Reply:
handleColorSelection will be passed as: `activateCallback`